client/src/main/java/org/apache/rocketmq/client/impl/producer/DefaultMQProducerImpl.java

```diff
@@ -622,14 +617,7 @@ private SendResult sendDefaultImpl(
                             default:
                                 break;
                         }
-                    } catch (RemotingException e) {
-                        endTimestamp = System.currentTimeMillis();
-                        this.updateFaultItem(mq.getBrokerName(), endTimestamp - beginTimestampPrev, true);
-                        log.warn(String.format("sendKernelImpl exception, resend at once, InvokeID: %s, RT: %sms, Broker: %s", invokeID, endTimestamp - beginTimestampPrev, mq), e);
-                        log.warn(msg.toString());
-                        exception = e;
-                        continue;
-                    } catch (MQClientException e) {
+                    } catch (RemotingException | MQClientException e) {
```

Comment:
Multi-catches are not supported at language level '6'

Reply:
ok,i had restore this  exception statement for jdk1.6